app/assets/images/chevron-black.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Would this work just as well with https://github.com/AusDTO/gov-au-beta/blob/develop/vendor/assets/images/icons/chevron-down--navy.svg ?